azext_iot/common/_azure.py

```diff
@@ -241,15 +241,21 @@ def _find_iot_dps_from_list(all_dps, dps_name):
 
 
 def get_iot_central_tokens(cmd, app_id, central_api_uri):
-    def get_event_hub_token(app_id, iotcAccessToken):
+    def formatUrl(url):
```

Comment:
I don't think this formatUrl is particularly helpful, it seems to just be adding https:// infront of the app-id if the app-id doesn't have https infront of it

But an app-id **should** never have https infront of it